Core/Money.cs

```diff
@@ -1,14 +1,14 @@
 namespace Core
 {
-    public struct Money
-    {
-        public Money(decimal amount, Currency currency)
-        {
-            Amount = amount;
-            Currency = currency;
-        }
+	public struct Money
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Replace all tab characters in this file by sequences of white-spaces.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=4931949)